kernelci/rootfs.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2019 Collabora Limited
+# Author: Lakshmipathi G <lakshmipathi.ganapathi@collabora.com>
+#
+# This module is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the Free
+# Software Foundation; either version 2.1 of the License, or (at your option)
+# any later version.
+#
+# This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this library; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+from kernelci import shell_cmd
+
+
+def build(name, config, debos_path):
+
+    for arch_type in config.arch_list:
+        cmd = 'cd {debos_path} && \
```

Comment:
I think it'd be better to use Python `os.chdir()` to switch the directory instead of calling `cd` from underlying shell.

Reply:
Added this change and will update this PR.